pkgs/development/python-modules/pymongo/default.nix

```
@@ -6,12 +6,12 @@
 
 buildPythonPackage rec {
   pname = "pymongo";
-  version = "4.1.1";
+  version = "4.2.0";
   disabled = pythonOlder "3.6";
```

No releases [below 3.7](https://github.com/mongodb/mongo-python-driver/blob/4.2.0/setup.py#L319) are supported.

```suggestion
  disabled = pythonOlder "3.7";
```